src/extensions/extensionsHelper.ts

```diff
@@ -85,7 +91,12 @@ export const AUTOPOULATED_PARAM_PLACEHOLDERS = {
   DATABASE_INSTANCE: "project-id-default-rtdb",
   DATABASE_URL: "https://project-id-default-rtdb.firebaseio.com",
 };
-export type ReleaseStage = "stable" | "alpha" | "beta" | "rc";
+export const resourceTypeToNiceName: Record<string, string> = {
```

Comment:
Exported consts usually are CAPS_CASE (https://google.github.io/styleguide/tsguide.html#identifiers-constants)

Reply:
This isn't part of my change, the order got flipped when merging I think. I can change it if you'd like.

Reply:
Ooops, was reviewing on autopilot. Feel free to, but certainly not blocking